src/components/Modal/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { Modal } from "./Modal";
+export type { ModalProps } from "./Modal";
```

Comment:
I missed this on my first pass. But, I believe we'll also need to re-export the rest of the Modal components; `ModalBody`, `ModalOverlay`, `ModalCloseButton`, etc.